Modules/ThirdParty/PNG/UpdateFromUpstream.sh

```diff
@@ -3,7 +3,7 @@
 thirdparty_module_name='PNG'
 
 upstream_git_url='git://git.code.sf.net/p/libpng/code'
-upstream_git_branch='libpng16'
+upstream_git_branch='libpng-1.6.31-signed'
```

Comment:
The update was done to 1.6.37+ version. Why change this here?

Reply:
Opps, that was my local change.

Reply:
fixed.

Reply:
This doesn't appear to have been reverted back to 1.6.37+ and in 029b2617a602413feda7c18325424c48dab03419 is still "libpng-1.6.31-signed".  The actual release of 1.6.37 could be found at this [tag](https://github.com/glennrp/libpng/tree/v1.6.37) in the officially approved git repo of the code that is otherwise hosted at the sourceforge link.

Reply:
I will probably get around to it by the end of the day, assuming no one beats me to fixing it with a new PR.

Reply:
PR #1694 has the fix.